bokehjs/src/lib/core/enums.ts

```diff
@@ -127,6 +127,9 @@ export type MutedPolicy = typeof MutedPolicy["__type__"]
 export const Orientation = Enum("vertical", "horizontal")
 export type Orientation = typeof Orientation["__type__"]
 
+export const OutlineShapeName = Enum("plain", "box", "rectangle", "square", "circle", "ellipse", "trapezium", "parallelogram", "diamond", "triangle")
```

Comment:
I would suggest "trapezoid" instead of "trapezium", and "none" rather than "plain" (plain what?)